src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.AgentPool.ps1

```diff
@@ -127,6 +127,7 @@ class AgentPool: ADOSVTBase
             {
                 if ($this.AgentPoolOrgObj.autoProvision -eq $true) {
                     $controlResult.AddMessage([VerificationResult]::Failed,"Auto-provisioning is enabled for the $($this.AgentPoolOrgObj.name) agent pool.");
+                    $controlResult.AdditionalInfoInCSV += "NA";
```

Comment:
Add NA for passing scenarios as well (just AdditionalInfoInCSV). Also do we have 'AdditionalInfo' in this control similar to the one you added in auto update control? If not we should add it here as well.